bin/3scale_backend_worker

```diff
@@ -10,7 +10,7 @@ end
 options = {
   multiple: true,
   dir_mode: :normal,
-  dir: '/var/run/3scale'
+  dir: "#{ENV['WORKER_PIDFILE_DIR']}"
```

Comment:
Why this variable? Can we be sure it's set in production?

This could be a breaking change. If you aren't sure, use:

```suggestion
  dir: "#{ENV['WORKER_PIDFILE_DIR'] || '/var/run/3scale'}"
```

On the other hand, `options` are only used for daemonizing. So I think all this `options` block and the `File.writable` block should run only when `--debug` is not given. A way to do this is to create a method for the debug mode that only calls `ThreeScale::Backend::Worker.work` and another method for the production mode that takes care of all this daemonize logic. And call one or another based on `--debug` presence.